packages/entities/entities-plugins/src/components/free-form/shared/ArrayField.vue

```diff
@@ -150,6 +155,8 @@ import { useField, useFieldAttrs, useFormShared, useItemKeys } from './composabl
 import useI18n from '../../../composables/useI18n'
 import * as utils from './utils'
 import Field from './Field.vue'
+import type { ArrayLikeFieldSchema } from '../../../types/plugins/form-schema'
+import TagField from './TagField.vue'
```

Comment:
Non-blocking: since `TagField` is no longer used exclusively for tags, we could consider renaming this component in the future. Other field components follow the `[ValueType]Field` naming pattern, so perhaps `StringArrayField` would be appropriate.